### DISCUSSION TITLE
Tags added by Clumio to AWS resources

### DISCUSSION THREAD

**Discussion by thevops** [clumio-code/terraform-clumio-aws-template#67] [Q&A] [answered]
What is the purpose of this tag - `"clumio.modifystreams.tag" = "disabled_to_new_image"` - added to DynamoDB table?

Could you provide a documentation including information about tags and other modifications done by Clumio to AWS resources?

**Comment by ayushverma-clumio** [comment, marked as answer]
I would like to inform you that, initially, tags were added to identify tables modified by Clumio as part of our backup process. However, based on valuable customer feedback, we have decided to discontinue this practice and will no longer add these tags to any customer's tables.

**Reply by thevops** [reply]
Thank you for the information.
